packages/booter-lb3app/src/lb3app.booter.ts

```diff
@@ -73,6 +73,17 @@ export class Lb3AppBooter implements Booter {
       });
     }
 
+    const models = lb3App.models;
+    if (models) {
+      const visited: unknown[] = [];
+      Object.keys(models).forEach(key => {
+        const model = models[key];
+        if (visited.includes(model)) return;
+        visited.push(model);
+        this.app.bind(`models.lb3-${key}`).to(model).tag('model');
```

Comment:
I am not sure if it's a good idea to use `models` namespace and prefix the name with `lb3-`. I am concerned about possible conflicts this could create once we start binding LB4 models too. Have you considered using a different namespace, e.g. `lb3-models`? Same comment applies on the tag. At the end of the day, LB3 models are not interchangeable with LB4 models, therefore it's better to clearly mark them as different artifacts.

```suggestion
        this.app.bind(`lb3-models.${key}`).to(model).tag('lb3-model');
```

Reply:
I agree; I'll update this. I referenced the [code for binding lb3 datasources](https://github.com/strongloop/loopback-next/blob/master/packages/booter-lb3app/src/lb3app.booter.ts#L65-L74) when I was doing this, so I can open a new PR to change that too.